src/relay/op/tensor/binary.cc

```diff
@@ -10,46 +10,36 @@
 namespace tvm {
 namespace relay {
 
-#define RELAY_REGISTER_BINARY_OP(OpName)                               \
-  TVM_REGISTER_API("relay.op._make." OpName)                           \
-  .set_body_typed<Expr(Expr, Expr)>([](Expr lhs, Expr rhs) {           \
-      static const Op& op = Op::Get(OpName);                           \
-      return CallNode::make(op, {lhs, rhs}, Attrs(), {});              \
-    });                                                                \
-  RELAY_REGISTER_OP(OpName)                                            \
-  .set_num_inputs(2)                                                   \
-  .add_argument("lhs", "Tensor", "The left hand side tensor.")         \
-  .add_argument("rhs", "Tensor", "The right hand side tensor.")        \
-  .add_type_rel("Broadcast", BroadcastRel)
+#define RELAY_MAKE_REGISTER_BINARY_OP(OpName) RELAY_REGISTER_BINARY_OP("relay.op._make.", OpName)
 
 // Addition
-RELAY_REGISTER_BINARY_OP("add")
+RELAY_MAKE_REGISTER_BINARY_OP("add")
```

Comment:
keep the original name and marcro

Reply:
what should be the name of RELAY_REGISTER_UNARY_OP and RELAY_MAKE_REGISTER_BINARY_OP? I dont think having them the same is a good idea

Reply:
I thought both name ae RELAY_REGISTER_UNARY_OP AND RELAY_REGISTER_BINARY_OP?

Reply:
Yes, but what should be the name of the macro in op_common? or should we just call it directly (without helper macro that wrap it)?